school.py

```diff
@@ -69,26 +43,70 @@ def setup_schools(args, net: sumolib.net.Net, stats: ET.ElementTree, school_coun
                   f"school_close_latest:\t {school_close_latest}\n\t"
                   f"school_stepsize:\t\t {school_stepsize}")
 
-    # Find edges to place schools on
-    new_school_edges = find_school_edges(net, number_new_schools, centre)
+    xml_schools = stats.find('schools')
+    if xml_schools is None:
+        xml_schools = ET.SubElement(stats.getroot(), "schools")
 
     # Insert schools, with semi-random parameters
-    logging.info("Inserting " + str(len(new_school_edges)) + " new school(s)")
-    for school in new_school_edges:
-        begin_age = random.randint(int(args["--schools.begin-age"].split(",")[0]),
-                                   int(args["--schools.begin-age"].split(",")[1]))
-        end_age = random.randint(int(args["--schools.end-age"].split(",")[0]) if begin_age + 1 <= int(
-            args["--schools.end-age"].split(",")[0]) else begin_age + 1,
-                                 int(args["--schools.end-age"].split(",")[1]))
-        logging.debug(f"Using begin_age: {begin_age}, end_age: {end_age} for school(s)")
+    logging.info("Inserting " + str(len(new_school_edges)) + f" new {school_type}(s)")
```

Comment:
Mixed string interpolation
```suggestion
    logging.info(f"Inserting {str(len(new_school_edges))} new {school_type}(s)")
```

Reply:
Fixed in 433d129035ebe74d19701c40291dee1be62ff5f4